letsencrypt/auth_handler.py

```diff
@@ -113,6 +114,26 @@ def _solve_challenges(self):
                     cont_resp = self.cont_auth.perform(self.cont_c)
                 if self.dv_c:
                     dv_resp = self.dv_auth.perform(self.dv_c)
+                    for achall, response in zip(self.dv_c, dv_resp):
+                        simple_verify_return = True
```

Comment:
BTW, it feels like all this stuff within `for` block should be moved to a separate function: `def simple_verify(self, achall, response)`. Testing should be easier.